Update: Added info re uppercase and lowercase

leebut opened the pull request:
Included information to explain that lowercase and camelCase can be used for const identifiers.

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #35059

leebut committed:
Update: Added info re uppercase and lowercase

Included information to explain that lowercase and camelCase can be used for const identifiers.

theNaut1lus reviewed (changes requested):
_Uppercase will be for immutable variables in the first line._
Or Instead of adding this note here we should add it on the challenge which originally had the problem.

co16353sidak commented on curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/declare-a-read-only-variable-with-the-const-keyword.english.md:
```diff
@@ -10,6 +10,8 @@ The keyword <code>let</code> is not the only new way to declare variables. In ES
 <blockquote>"use strict"<br>const FAV_PET = "Cats";<br>FAV_PET = "Dogs"; // returns error</blockquote>
 As you can see, trying to reassign a variable declared with <code>const</code> will throw an error. You should always name variables you don't want to reassign using the <code>const</code> keyword. This helps when you accidentally attempt to reassign a variable that is meant to stay constant. A common practice when naming constants is to use all uppercase letters, with words separated by an under ...
+  
+  **Note:** It is common for developers to use uppercase variable identifiers for mutable values and lowercase or camelCase for mutable values (objects and arrays). In a later challenge you will see an example of a lowercase variable identifier being used for an array.
```
It should be immutable instead of mutable in the first line.

leebut replied:
Thank you. Changed it.

leebut committed:
Fix: corrected mutable to immutable

I corrected the error that uppercase is used for mutable values.

Manish-Giri reviewed (changes requested):
@leebut I tested your change locally. Using `**Note:**` won't produce the effect you're looking for on the "Note" -

![1](https://user-images.githubusercontent.com/11348778/53012773-f9b52f00-3411-11e9-99cf-5c2ee42047c2.png)

Replace `**Note:**` with `<strong>Note:</strong>` - 

![2](https://user-images.githubusercontent.com/11348778/53012843-26694680-3412-11e9-9c84-5f1ffe98a26c.png)

leebut committed:
Fix: changed ** ** to <strong></strong>

I changed the markdown **  ** to <strong></strong>, as I was informed that ** didin't work.

leebut commented:
Thank you for your correction.

I read on the [Mastering Markdown](https://guides.github.com/features/mastering-markdown/) page that  ** ** was the Github standard markdown for bold text. It looked fine in the preview. Anyway, I've changed it. Thanks.

RandellDawson commented:
@leebut FYI - The Challenge's description and instruction sections do not use markdown for formatting.  Only the Guide articles make use of markdown for displaying on the site.

leebut commented:
Okay. Thank you for the clarification. I hope my mistakes aren't annoyances.

RandellDawson commented:
@leebut No, it is not.  I got confused when first learning the differences between working on guide articles formatting vs. curriculum challenges formatting.  It is mentioned in the documentation, but may the documentation for the curriculum should have a note that markdown formatting does not work for those sections.

Manish-Giri reviewed (approved):
LGTM 💯

RandellDawson reviewed (approved):
I have tested this locally and everything look good.

@Manish-Giri Since the earlier PR for this file does make changes to the same section, I think it is fine to go ahead and merge this one out of sequence.

Manish-Giri reviewed (commented):
@RandellDawson All right!

@leebut Thank you for your contribution to the page! 👍
